src/Interpreters/BlobStorageLog.h

```diff
@@ -0,0 +1,100 @@
+#pragma once
+
+
+#include <Interpreters/SystemLog.h>
+#include <Core/NamesAndTypes.h>
+#include <Core/NamesAndAliases.h>
+#include <Poco/Message.h>
+#include <chrono>
+
+namespace Aws::S3
+{
+    class S3Error;
+}
+
+namespace DB
+{
+
+struct BlobStorageLogElement
```

Comment:
Is it possible to:
1. add the compressed size in bytes?
2. add a new event_type DeleteFailed in case of a blob has been not removed from s3 due to some errors.

Reply:
Related test https://github.com/ClickHouse/ClickHouse/blob/0f9acd685ab730a135af8d6d25a474142dc47fa7/tests/integration/test_merge_tree_s3/test.py#L304